lib/connectors/memory.js

```diff
@@ -461,6 +460,15 @@ function applyFilter(filter) {
     if (typeof value === 'string' && (example instanceof RegExp)) {
       return value.match(example);
     }
+
+    if (example.regexp) {
+      var normalizedRegExp = utils.toRegExp(example.regexp);
+      if (normalizedRegExp instanceof Error)
```

Comment:
That's not good. You need to dig into the root cause. The idea is that `dao` will normalize the operand of `regexp` to an instance of `RegExp` so that connectors don't have to deal with conversions.